book/page_creation.rst

```diff
@@ -137,7 +137,7 @@ You can even shorten this with the handy :class:`Symfony\\Component\\HttpFoundat
     // --> don't forget this new use statement
     use Symfony\Component\HttpFoundation\JsonResponse;
 
-    class LuckyController
+    class LuckyController extends Controller
```

Comment:
well, this example does not depend on any method of the base class. So why extending it ?